src/main/java/properties/ReadProperties.java

```diff
@@ -43,12 +44,25 @@ private void readPropertyFile(String file) {
     }
 
     /***
-     * Receives the key value from config.properties
-     * @param key key from config.properties
+     * Receives the key value from property files
+     * @param key key from property files
      * @return key value
      */
-    public String getKeyProperty(String propsFileName, String key) {
-        readPropertyFile(propsFileName);
-        return propsMap.get(propsFileName).getProperty(key);
+    public String getKeyProperty(String environment, String key) {
```

Comment:
This method is not reusable. You store it in a common class ReadProperties but it can read ONLY environment configs. If you add a different property file you will create another method for reading property? 
What's different between an environment property file and any property file? Nothing apart from the name. Why do you need a separate method for that?

Also Selecting which environment is active (AKA which property file to use) is NOT part of ReadProperties class but part of Environment config. ReadProperties is used ONLY to load properties from a property file without any other actions.

Reply:
Understood and fixed